src/halmos/__main__.py

```diff
@@ -238,17 +239,26 @@ def mk_solver(args: HalmosConfig, logic="QF_AUFBV", ctx=None):
     )
 
 
+# NOTE: Do NOT remove the `con_addr()` wrapper for TEST_TARGET.
+# The return type should be BitVecSort(160) as it is used as a key for ex.code.
+# The keys of ex.code are compared using structural equality with other BitVecRef addresses.
+TEST_TARGET = con_addr(FOUNDRY_TEST)
+TEST_CALLER = FOUNDRY_CALLER
+TEST_ORIGIN = FOUNDRY_ORIGIN
```

Comment:
why introduce aliases instead of just using the `FOUNDRY_*` constants?

Reply:
ah just wanted to use the same prefix for all three.  we can also change FOUNDRY_TEST to be `con_addr(0x...)`, and use the `FOUNDRY_*` prefix.  do you prefer that?

Reply:
yeah that works! 👍

Reply:
https://github.com/a16z/halmos/pull/472/commits/b338f2a1a5c87b79d51a13d443db54979dc49955